tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -104,6 +104,37 @@ public:
       fBranches.clear();
       fNames.clear();
    }
+
+   void AssertNoNullBranchAddresses()
+   {
+      std::vector<TBranch *> branchesWithNullAddress;
+      std::copy_if(fBranches.begin(), fBranches.end(), std::back_inserter(branchesWithNullAddress),
+                   [](TBranch *b) { return b->GetAddress() == nullptr; });
+
+      if (branchesWithNullAddress.empty())
+         return;
+
+      // otherwise build error message and throw
+      using namespace std::string_literals;
```

Comment:
What is the usage of the namespace here?

Reply:
See "Notes" at https://en.cppreference.com/w/cpp/string/basic_string/operator%22%22s

Reply:
Ok thanks, it's just that I don't see usage of `operator""s` in the code

Reply:
ah, sorry I did not get your first comment. It's a leftover, now removed, thanks.